packages/esm-patient-medications-app/src/components/medications-details-table.component.tsx

```diff
@@ -365,99 +365,100 @@ function OrderBasketItemActions({
-          valueCoded: medication.doseUnits?.uuid,
+        drug: medication.drug,
+        duration: medication.duration,
+        durationUnit: {
+          value: medication.durationUnits?.display,
+          valueCoded: medication.durationUnits?.uuid,
         },
+        freeTextDosage:
+          medication.dosingType === 'org.openmrs.FreeTextDosingInstructions' ? medication.dosingInstructions : '',
         frequency: {
-          valueCoded: medication.frequency?.uuid,
           value: medication.frequency?.display,
+          valueCoded: medication.frequency?.uuid,
         },
-        route: {
-          valueCoded: medication.route?.uuid,
-          value: medication.route?.display,
-        },
-        commonMedicationName: medication.drug?.display,
+        indication: medication.orderReasonNonCoded,
         isFreeTextDosage: medication.dosingType === 'org.openmrs.FreeTextDosingInstructions',
-        freeTextDosage:
-          medication.dosingType === 'org.openmrs.FreeTextDosingInstructions' ? medication.dosingInstructions : '',
+        numRefills: medication.numRefills,
+        orderer: medication.orderer.uuid,
         patientInstructions:
           medication.dosingType !== 'org.openmrs.FreeTextDosingInstructions' ? medication.dosingInstructions : '',
-        asNeeded: medication.asNeeded,
-        asNeededCondition: medication.asNeededCondition,
-        startDate: medication.dateActivated,
-        duration: medication.duration,
-        durationUnit: {
-          valueCoded: medication.durationUnits?.uuid,
-          value: medication.durationUnits?.display,
-        },
         pillsDispensed: medication.quantity,
-        numRefills: medication.numRefills,
-        indication: medication.orderReasonNonCoded,
-        orderer: medication.orderer.uuid,
-        careSetting: medication.careSetting.uuid,
+        previousOrder: null,
         quantityUnits: {
           value: medication.quantityUnits?.display,
           valueCoded: medication.quantityUnits?.uuid,
         },
+        route: {
+          value: medication.route?.display,
+          valueCoded: medication.route?.uuid,
+        },
+        unit: {
+          value: medication.doseUnits?.display,
+          valueCoded: medication.doseUnits?.uuid,
+        },
+        uuid: medication.uuid,
       },
     ]);
     openOrderBasket();
   }, [items, setItems, medication, openOrderBasket]);
 
   const handleModifyClick = useCallback(() => {
     const newItem: DrugOrderBasketItem = {
-      uuid: medication.uuid,
-      display: medication.drug?.display,
-      previousOrder: medication.uuid,
-      startDate: new Date(),
       action: 'REVISE',
-      drug: medication.drug,
+      asNeeded: medication.asNeeded,
+      asNeededCondition: medication.asNeededCondition,
+      careSetting: medication.careSetting?.uuid,
+      commonMedicationName: medication.drug?.display,
+      dateActivated: medication?.dateActivated,
```

Comment:
Key change: renaming this property from `startDate` to `dateActivated` and setting it's value to the existing medication's `dateActivated` property.